schemas/validate_manifest.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Change the name of the schema file to `manifest_v1_schema.json` and move the `schemas` folder into the `dpypelines` directory.